HOWTO_RELEASE.md

```diff
@@ -0,0 +1,169 @@
+  ```
+  $ rm build/ -rf
+  $ python setup.py sdist
+  $ python setup.py bdist_wheel
+  ```
+  The packages are by default stored in a `dist` folder.
+
+- To build the conda packages, you should *not* work in a specific environment but rather exit to the root environment. There, install the `conda-build` tool for building packages:
+  ```
+  $ source deactivate
+  $ conda install conda-build
+  ```
+- Invoke the following command to build a package for your platform and all supported Pyhton versions:
+  ```
+  $ conda build conda/ --python 2.7
+  $ conda build conda/ --python 3.4
+  $ conda build conda/ --python 3.5
+  ...
+  ```
+- Assuming this succeeds, enter the directory one above where the conda package was stored (as printed in the output), e.g.,
+  ```
+  $ cd $HOME/miniconda3/conda-bld/
+  ```
+  There, for each Python version "translate" the package to all platforms since ODL is actually platform-independent:
+  ```
+  $ conda convert --platform all <package>
+  ```
+  Replace `<package>` by the package file as built by the previous `conda build` command.
+
+## 6. Test installing the local packages and check them
+
+Before actually uploading packages to "official" servers, first install the local packages and run the unit tests.
+
+- Install directly from the source package (`*.tar.gz`) or the wheel (`*.whl`) into a new conda environment:
+  ```
+  $ source deactivate
+  $ conda create -n pypi_install python=X.Y  # choose Python version
+  $ source activate pypi_install
+  $ cd /path/to/odl_repo
+  $ pip install dist/<pkg_filename>
+  $ python -c "import odl; odl.test()"
+  ```
+- Install and test the local conda packages in a new conda environment:
+  ```
+  $ source deactivate
+  $ conda create -n conda_install python=X.Y  # choose Python version
+  $ source activate conda_install
+  $ conda install --use-local nomkl odl
+  $ python -c "import odl; odl.test()"
+  ```
+
+## 7. Upload the packages to the official locations
+
+Installing the packages works, now it's time to put them out into the wild.
+
+- Install the `twine` package for uploading packages to PyPI in your working environment:
+  ```
+  $ source deactivate
+  $ source activate release35
+  $ pip install twine
+  ```
+- Upload the source package and the wheel to the PyPI server using `twine`:
+  ```
+  $ cd /path/to/odl_repo
+  $ twine upload -u odlgorup dist/<pkg_filename>
+  ```
+  This requires the access credentials for the `odlgroup` user on PyPI.
+- Upload the conda packages to the `odlgroup` channel in the Anaconda cloud. The upload requires the `anaconda-client` package:
+  ```
+  $ conda install anaconda-client
+  $ cd $HOME/miniconda3/conda-bld
+  $ anaconda upload -u odlgroup `find . -name "odl-X.Y.Z*"`
+  ```
+  For this step, you need the access credentials for the `odlgroup` user on the Anaconda server.
+
+## 8. Bump current `master` to a development version
```

Comment:
This should be done before the conda stuff in order to avoid misstaken versions. Frankly, it should be done after the release branch is created.